xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/test/java/org/xwiki/rest/templates/RestExceptionPageTest.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.rest.templates;
+
+import java.io.IOException;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.junit.jupiter.api.Test;
+import org.xwiki.template.TemplateManager;
+import org.xwiki.test.page.PageTest;
+import org.xwiki.velocity.VelocityManager;
+import org.xwiki.velocity.tools.EscapeTool;
+
+import static org.apache.commons.lang3.exception.ExceptionUtils.getStackTrace;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Tests of the rest Velocity templates.
+ *
+ * @version $Id$
+ * @since 13.8RC1
+ * @since 13.7.1
+ * @since 13.4.4
+ */
+class RestExceptionPageTest extends PageTest
```

Comment:
Note that it's `REST` and not `Rest` so all classes should use `REST`. Is it because we've used `Rest` in the past in some class names that you've continued?

Reply:
Answering my question: we have `XWikiRestComponent` already for example...

Reply:
Hard to be consistent now. Same for `HTTP` (my preference) over `Http`. I thought we agreed to use the uppercase form a long time but either it wasn't the case or we didn't pay attention.